module/s3-scan-object/app.js

```diff
@@ -178,7 +178,12 @@ const getRoleCredentials = async (stsClient, roleArn) => {
   try {
     const command = new AssumeRoleCommand({ RoleArn: roleArn, RoleSessionName: "s3-scan-object" });
     const response = await stsClient.send(command);
-    credentials = response.Credentials;
+    // Submitted, without comment or judgement
```

Comment:
code archeologists will be forever confused.

Reply:
hah, and this was the sanitized code comment